Kanban/Services/UserService.cs

```diff
@@ -32,5 +32,36 @@ public async Task<List<User>> GetUsers()
 
             return success ? user : null;
         }
+
+        public async Task<bool?> AnonymizeUser(int id)
+        {
+            User? user = await userRepository.GetUserById(id);
+
+            if (user != null)
+            {
+                user.Name = string.Empty;
+                bool success = await userRepository.UpdateUser(user);
+                return success;
+            }
+            else
+            {
+                return null;
+            }
+        }
+
+        public async Task<bool?> DeleteUser(int id)
```

Comment:
What are your considerations for the return type here?

Reply:
Yes, it does.

But pretend you are just looking at the signature (or the interface), and someone else returned a `bool?` from a `DeleteUser` method. What would it do?

(If you want a three-way result pattern, then maybe you should use a custom enum to signify the result?)